templates/taxi/index.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% block content %}
+  <h1>Taxi service</h1>
+  <p>
+    Welcome to our taxi service<br>
+    The best drivers and cars you will find here
+  </p>
+  <ul>
+    <li>Number of cars: {{ num_cars }}</li>
+    <li>Number of drivers: {{ num_cars }}</li>
```

Comment:
There is a mistake here. The template is displaying `num_cars` for both the number of cars and the number of drivers. It should display `num_drivers` for the number of drivers.

Reply:
Done